.gitignore

```diff
@@ -67,6 +67,9 @@ docs/.static/api/
 # rvm
 .ruby-gemset
 
+# rbenv
```

Comment:
This is not related to the pr, but decided to keep it here because it's a small change.